test/e2e/cordon-only-test

```diff
@@ -42,18 +47,30 @@ set -x
 helm "${anth_helm_args[@]}"
 set +x
 
+aemm_helm_args=(
+  upgrade
+  --install
+  $CLUSTER_NAME-aemm
+  https://github.com/aws/amazon-ec2-metadata-mock/releases/download/v"$AEMM_VERSION"/amazon-ec2-metadata-mock-"$AEMM_VERSION".tgz
+  --namespace default
+  --set servicePort="$IMDS_PORT"
+)
+
+set -x
+helm "${aemm_helm_args[@]}"
```

Comment:
Yeah, it should once I update the AEMM daemonset to support Windows. I didn't include `common_helm_args` in this PR because neither args (targetNodeOS and nodeSelector) are applicable to AEMM today